### ISSUE TITLE
Incorrect formatting for graph query

### ISSUE THREAD

**Issue by ioannist** [surrealdb/surrealist#272]
**Describe the bug**
The following query has its formatting messed up

`SELECT <-mb_hasInBlock FROM mb_relation:['parachainstaking','topdelegations', NONE, NONE, NONE]..=['parachainstaking','topdelegations'] LIMIT 1000;
`

**To Reproduce**
Just paste the above query to surrealist

**Expected behavior**
Color formatting should be correct

**Environment:**
```
Platform: Desktop
OS: Windows_NT
Architecture: x86_64
WebView: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.0.0
Version: 2.0.6
Flags: featureFlags: false, models_view: true, apidocs_view: true, themes: false, newsfeed: true

```

**Comment by kearfy** [comment]
Idiom paths are currently not supported by the grammar as implementing them caused a number of issues. I need to find some time to properly look at it again. Thanks for reporting!

**Comment by macjuul** [comment]
This has been resolved in the 2.1 release
